ros2_kdl_node_1a:
  ros__parameters:

    traj_duration: 1.5
    acc_duration: 0.5
    total_time: 1.5
    trajectory_len: 150

    Kp: 5.0

    end_position_x: 0.8
    end_position_y: 0.5
    end_position_z: 0.4

ros2_kdl_node_1b:
  ros__parameters:

    traj_duration: 1.5
    acc_duration: 0.5
    total_time: 1.5
    trajectory_len: 150

    Kp: 5.0

    end_position_x: 0.8
    end_position_y: 0.5
    end_position_z: 0.4

ros2_kdl_node_1c:
  ros__parameters:

    traj_duration: 1.5
    acc_duration: 0.5
    total_time: 1.5
    trajectory_len: 150

    Kp: 5.0
